src/nupic/regions/ScalarSensor.hpp

```diff
@@ -67,8 +67,7 @@ class ScalarSensor : public RegionImpl, Serializable {
   virtual std::string executeCommand(const std::vector<std::string> &args,
                                      Int64 index) override;
 
-  virtual size_t
-  getNodeOutputElementCount(const std::string &outputName) const override;
+  virtual Dimensions askImplForOutputDimensions(const std::string &name) override;
```

Comment:
can we have doc explaining what the name can be? "bucket","encoded" as seen in cpp

Reply:
> can we have doc explaining what the name can be? "bucket","encoded" as seen in cpp

Well, that is one of the purposes of the Spec.
But you are right, a new set of docs in the wiki that describes the entire API would be very useful.